### ISSUE TITLE
Load application controller before any other controller

### ISSUE THREAD

**Issue by Nephos** [Amber-Crystal/amber#134]
Hello,

I add a problem when generating a controller with a name that comes before "application_controller.cr".
The default loader could be like

```crystal
class ApplicationController < Amber::Controller::Base
  LAYOUT = "application.slang"
end

require "./**"
```

and in the init could be

```crystal
require "amber"
require "./controllers/application_controller"
# ...
```

What do you think ?

**Comment by elorest** [comment]
Yeah I didn't think about what would happen in this case if you had a controller that alphabetically proceeded `application_controller`. This is an issue with amber_cmd however.

**Comment by Nephos** [comment]
Sure !

**Comment by gabrielrios** [comment]
Isn't this fixed on Amber-Crystal/amber_cmd#55?

**Comment by elorest** [comment]
Yes. Thanks it should have been closed here.
